PennMobile/LaundryPrototype/Models/Laundry.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  Laundry.swift
+//  PennMobile
+//
+//  Created by Nathan Aronson on 10/5/25.
+//  Copyright © 2025 PennLabs. All rights reserved.
+//
+
+struct LaundryHallId: Codable, Hashable {
+    let name: String
+    let hallId: Int
+    let location: String
+    
+    enum CodingKeys: String, CodingKey {
+        case name
+        case hallId = "hall_id"
+        case location
+    }
```

Comment:
You don't need to manually specify `CodingKeys` if you set `keyDecodingStrategy` on your `JSONDecoder` to `.convertFromSnakeCase`